packages/flutter/lib/src/rendering/binding.dart

```diff
@@ -60,6 +60,27 @@ abstract class RendererBinding extends BindingBase implements SchedulerBinding,
           return _forceRepaint();
         }
       );
+      registerBoolServiceExtension(
+          name: 'debugPaintBaselinesEnabled',
+          getter: () async => debugPaintBaselinesEnabled,
+          setter: (bool value) async {
```

Comment:
please make the first two of these use the same basic structure (either `async` or not, I trust your judgement). (note that the third one is intentionally a bit different)